src/app/calendar/ParcelCalendar.tsx

```diff
@@ -13,20 +22,67 @@ const Centerer = styled.div`
 
 const CalendarWrapper = styled.div`
     width: 100vmin;
-    margin: 2em;
+    max-width: 1100px;
+    margin: 0 2em 2em;
 `;
 
-interface ParcelCalendarProps {
-    events: CalendarEvent[];
-}
-
 const ParcelCalendar: React.FC<ParcelCalendarProps> = (props) => {
+    const chosenTheme = useTheme();
+
+    const colorMap: LocationColorMap = {
+        "Brixton Hill - Methodist Church": {
+            color: `${chosenTheme.rainbow.lightRed.background}`,
+            text: `${chosenTheme.rainbow.lightRed.foreground}`,
+        },
+        "Clapham - St Stephens Church": {
+            color: `${chosenTheme.rainbow.lightOrange.background}`,
+            text: `${chosenTheme.rainbow.lightOrange.foreground}`,
+        },
+        "N&B - Emmanuel Church": {
+            color: `${chosenTheme.rainbow.lightYellow.background}`,
+            text: `${chosenTheme.rainbow.lightYellow.foreground}`,
+        },
+        "Streatham - Immanuel & St Andrew": {
+            color: `${chosenTheme.rainbow.lightGreen.background}`,
+            text: `${chosenTheme.rainbow.lightGreen.foreground}`,
+        },
+        "Vauxhall Hope Church": {
+            color: `${chosenTheme.rainbow.darkGreen.background}`,
+            text: `${chosenTheme.rainbow.darkGreen.foreground}`,
+        },
+        "Waterloo - Oasis": {
+            color: `${chosenTheme.rainbow.lightBlue.background}`,
+            text: `${chosenTheme.rainbow.lightBlue.foreground}`,
+        },
+        "Waterloo - St George the Martyr": {
+            color: `${chosenTheme.rainbow.darkBlue.background}`,
+            text: `${chosenTheme.rainbow.darkBlue.foreground}`,
+        },
+        "Waterloo - St Johns": {
+            color: `${chosenTheme.rainbow.lightPurple.background}`,
+            text: `${chosenTheme.rainbow.lightPurple.foreground}`,
+        },
+        Delivery: {
+            color: `${chosenTheme.rainbow.darkPurple.background}`,
+            text: `${chosenTheme.rainbow.darkPurple.foreground}`,
+        },
+        default: {
+            color: `${chosenTheme.rainbow.lightBrown.background}`,
+            text: `${chosenTheme.rainbow.lightBrown.foreground}`,
+        },
+    };
```

Comment:
Also confused by this, surely it's easier to focus on getting themes merged if that is the most important bit then coming back and working on calendar after. Will mean you have to do less themes work as others will pick the colours from the theme to start with, but atm I am still using the original colours which will have to be changed later.